src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/GetApprenticeshipsResponse.cs

```diff
@@ -20,6 +20,7 @@ public class ApprenticeshipDetailsResponse
             public string Uln { get; set; }
             public string EmployerName { get; set; }
             public string ProviderName { get; set; }
+            public long Ukprn { get; set; }
```

Comment:
I would suggest referring to this as `ProviderId` throughout, to be consistent with the rest of the api.

Reply:
Renamed 👍